ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -399,13 +403,13 @@ tests:
       IP_STACK: v4
       NUM_EXTRA_WORKERS: "1"
     workflow: assisted-baremetal-operator-capi
-- as: e2e-ai-operator-ztp-ipv4v6-sno-ocp-49
```

Comment:
Lets create new ones instead of replacing 4.9 jobs? These are optional anyway

Reply:
but we do run those on changes, so if leaving 4.9 we should do them ``always_run: false`` to reduce costs